integration/eks-test/scripts/eks-cleanup.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Cleanup EKS cluster used for integration test.
+
+source ./integration/eks-test/scripts/eks-common.sh
+
+# Delete service and namespace from cluster 1 & 2
+$KUBECTL_BIN config use-context $EXPORT_CLS
+$KUBECTL_BIN delete svc $SERVICE -n $NAMESPACE
+
+# Verfication to check if there are hanging ServiceExport or ServiceImport CRDs and clears the finalizers to allow cleanup process to continue
+for CRD in $($KUBECTL_BIN get crd -n $NAMESPACE | grep multicluster | cut -d " " -f 1 | xargs); do 
+    $KUBECTL_BIN patch crd -n $NAMESPACE $CRD --type merge -p '{"metadata":{"finalizers": [null]}}';
```

Comment:
Was this the problem with eks cleanup?

Reply:
Yes, leftover finalizers were making the CRDs unable to be properly cleaned up, and so the solution was to remove the finalizers, then remove the CRDs.